src/main/java/com/mraof/minestuck/item/MSItems.java

```diff
@@ -721,6 +725,8 @@ public static void registerItems(final RegistryEvent.Register<Item> event)
 		registry.register(new WeaponItem(ItemTier.IRON, 4, -2.8F, 2.0F, MSItemTypes.MISC_TOOL, new Item.Properties().group(MSItemGroup.WEAPONS)).setRegistryName("metal_bat"));
 		registry.register(new WeaponItem(new WeaponItem.Builder(ItemTier.IRON, 5, -2.8F).efficiency(2.0F).set(MSItemTypes.MISC_TOOL).add(OnHitEffect.RAGE_STRENGTH, OnHitEffect.playSound(() -> MSSoundEvents.ITEM_HORN_USE, 1.5F, 1)), new Item.Properties().defaultMaxDamage(2048).group(MSItemGroup.WEAPONS)).setRegistryName("clown_club"));
 		registry.register(new WeaponItem(ItemTier.WOOD, 5, -2.8F, 2.0F, MSItemTypes.MISC_TOOL, new Item.Properties().defaultMaxDamage(100).group(MSItemGroup.WEAPONS)).setRegistryName("spiked_club"));
+		registry.register(new WeaponItem(ItemTier.IRON, 5, -2.8F, 2.0F, MSItemTypes.MISC_TOOL, new Item.Properties().group(MSItemGroup.WEAPONS)).setRegistryName("mace"));
+		registry.register(new WeaponItem(ItemTier.IRON, 6, -2.8F, 2.0F, MSItemTypes.MISC_TOOL, new Item.Properties().group(MSItemGroup.WEAPONS)).setRegistryName("m_ace"));
```

Comment:
We're transitioning to the builder constructor. Use that one instead of the deprecated constructor when adding new weapons.

Reply:
I think I did it right, but just tell me if not!